OpenRA.Mods.Common/Traits/Captures.cs

```diff
@@ -20,7 +20,7 @@ namespace OpenRA.Mods.Common.Traits
 	[Desc("This actor can capture other actors which have the Capturable: trait.")]
 	public class CapturesInfo : ConditionalTraitInfo
 	{
-		[Desc("Types of actors that it can capture, as long as the type also exists in the Capturable Type: trait.")]
+		[Desc("ProductionTypes of actors that it can capture, as long as the type also exists in the Capturable Type: trait.")]
```

Comment:
Wait, what?

Reply:
Oh, fuck. I guess I renamed stuff that I shouldn't have. Well, no bother I will just roll it back in those files and move on.
Another reason why I should squash the commits as last step.